src/LighthouseServiceProvider.php

```diff
@@ -38,25 +36,33 @@ class LighthouseServiceProvider extends ServiceProvider
     /**
      * Bootstrap any application services.
      *
+     * @param  \Illuminate\Validation\Factory  $validationFactory
      * @return void
      */
-    public function boot(): void
+    public function boot(ValidationFactory $validationFactory): void
     {
-        $this->mergeConfigFrom(__DIR__.'/../../config/config.php', 'lighthouse');
+        $this->mergeConfigFrom(__DIR__.'/../config/config.php', 'lighthouse');
 
         $this->publishes([
-            __DIR__.'/../../config/config.php' => config_path('lighthouse.php'),
+            __DIR__.'/../config/config.php' => config_path('lighthouse.php'),
```

Comment:
`config_path` helper is only used here. 
Let's remove the global helper and just do the stuff in the class itself.

Reply:
https://github.com/nuwave/lighthouse/pull/647